src/concepts/segments.md

```diff
@@ -1,75 +1,106 @@
-# Segments (beta)
+# User states/segments
 
-A segment is a group of clients who fit a set of criteria at a point in time. 
-The set of criteria itself can also be referred to as a "segment".
+A user state is a group of clients who fit a set of criteria at a point in time. 
+The set of criteria itself can also be referred to as a "user state".
 
-Typically you'll use segments to gain more insight into what is going on, by asking 
+In data science these user states would normally be called "segments"; we have been told to avoid that term to avoid confusion with marketing's segments.
```

Comment:
We've already made clear that user states and segments are more or less synonymous.  I would argue that this sentence is unnecessary, potentially divisive, and can be removed.

Reply:
This is the only place in the document where we explicitly say that "user states" and what DSes call "segments" are the same thing. I'll remove the reason we went with "user states" unless you can think of an innocuous phrasing.